From: Nigel Jones <nigelj@gmail.com>
Forwarded: Yes
Last-Update: 2011-08-06
Description: Documents lower case option for dsc option
 There were parsing problems with DSC option and, together with a fix
 to #176012 and #76569, the lower case of dsc was allowed.

Index: html2ps-1.0b7/html2ps.1
===================================================================
--- html2ps-1.0b7.orig/html2ps.1	2011-01-10 02:41:56.000000000 -0200
+++ html2ps-1.0b7/html2ps.1	2011-01-10 02:44:39.000000000 -0200
@@ -107,7 +107,7 @@
 Generate debugging information. You should always use this
 option when reporting problems with html2ps.
 .TP
-.B -D --DSC 
+.B -D --dsc --DSC 
 Generate DSC compliant PostScript. This requires Ghostscript and
 can take quite some time to do. Note that a PostScript file generated
 with this option cannot be used as input to html2ps for reformatting
